features/zoeken/met-geslachtsnaam-en-geboortedatum/overzicht.feature

```diff
@@ -0,0 +1,226 @@
+      Dan wordt alleen 'Jan' gevonden
+
+      Voorbeelden:
+        | geslachtsnaam | voorvoegsels |
+        | Maassen       | van          |
+        | maassen       | VAN          |
+        | MAASSEN       | Van          |
+
+  Regel: de optionele 'geslacht' parameter (niet hoofdlettergevoelig) kan worden gebruikt om de zoek criteria aan te scherpen
+
+    Abstract Scenario: geslachtsnaam, geslacht en geboortedatum van personen komen exact overeen met de opgegeven geslachtsnaam, geslacht en geboortedatum
+      Gegeven de persoon 'Aedel' met burgerservicenummer '000000027'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsaanduiding (04.10) |
+        |              19830526 | V                           |
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam 'Aedel' en geboortedatum '1983-05-26' en parameters
+        | geslacht              |
+        | <geslachtsaanduiding> |
+      Dan wordt alleen 'Aedel' gevonden
+
+      Voorbeelden:
+        | geslachtsaanduiding |
+        | v                   |
+        | V                   |
+
+  Regel: de optionele 'inclusiefOverledenPersonen' parameter moet worden opgegeven om overleden personen te kunnen vinden
+
+    Scenario: geslachtsnaam en geboortedatum van een overleden persoon komen exact overeen en de optionele 'inclusiefOverledenPersonen' parameter is niet opgegeven
+      Gegeven de persoon 'Jansen' met burgerservicenummer '000000028'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) |
+        |              19830526 |
+      En 'Jansen' is overleden
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam 'Jansen' en geboortedatum '1983-05-26'
+      Dan worden er geen personen gevonden
+
+    Scenario: geslachtsnaam en geboortedatum van een overleden persoon komen exact overeen en de optionele 'inclusiefOverledenPersonen' parameter is opgegeven
+      Gegeven de persoon 'Jansen' met burgerservicenummer '000000028'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) |
+        |              19830526 |
+      En 'Jansen' is op 23-03-2023 overleden
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam 'Jansen' en geboortedatum '1983-05-26' en parameters
+        | inclusiefOverledenPersonen |
+        | true                       |
+      Dan wordt 'Jansen' gevonden
+      En heeft 'Jansen' de volgende 'opschortingBijhouding' gegevens
+        | reden.code | reden.omschrijving | datum.datum | datum.langFormaat | datum.type |
+        | O          | overlijden         |  2023-03-23 |     23 maart 2023 | Datum      |
+
+  Regel: de optionele 'gemeenteVanInschrijving' parameter kan worden gebruikt om de zoek criteria aan te scherpen
+
+    Scenario: geslachtsnaam, geboortedatum van personen komen exact overeen met de opgegeven geslachtsnaam, geboortedatum maar de personen staan niet ingeschreven in de opgegeven gemeente
```

Comment:
in dit scenario:
- wordt gemeentecode opgegeven bij het adres, maar er wordt niks gedaan daarmee
- is de gemeentecode in het adres anders dan de gemeente van inschrijving

dat is heel nuttig als je de implementatie wilt documenteren (de implementatie krijgt naar gemeente van inschrijving en niet naar gemeentecode), maar verwarrend als je voor een afnemer de werking van de API wilt uitleggen.